shell/client/_topbar.scss

```diff
@@ -119,6 +119,7 @@ body>.topbar {
       background-size: 24px 24px;
       color: transparent;  // hide text
       width: 32px;
+      height: 32px;
```

Comment:
Many other places in this file have `height: 32px`, including `body>.topbar` itself. I propose that we tidy such things up in a separate pull request.